core/src/main/scala/kafka/controller/PartitionLeaderSelector.scala

```diff
@@ -134,9 +133,10 @@ class ReassignedPartitionLeaderSelector(controllerContext: ControllerContext) ex
  * New isr = current isr;
  * Replicas to receive LeaderAndIsr request = assigned replicas
  */
-class PreferredReplicaPartitionLeaderSelector(controllerContext: ControllerContext) extends PartitionLeaderSelector
-with Logging {
-  this.logIdent = "[PreferredReplicaPartitionLeaderSelector]: "
+class PreferredReplicaPartitionLeaderSelector(controllerContext: ControllerContext)
+  extends PartitionLeaderSelector
+    with Logging {
```

Comment:
This is just because of convenience of the programmer. Human eyes are much faster in finding something specific when it is aligned (see newspapers). So looking for some text in
`class A(b:C) extends D with E with F`
is much harder than in
```
class A (b: C) 
 extends D
    with E
    with F
```